### PR TITLE
Make level 1

### PR DESCRIPTION
**Pull request by DenisCo1100100**

### CODE REVIEW

File: DevIncubator_Autopark/Program.cs [hunk 1]
```diff
@@ -4,9 +4,44 @@ namespace DevIncubator_Autopark
...
+
+			averageCoefficient /= vehicles.Length;
+			Console.WriteLine($"Average tax coefficient = {averageCoefficient:0.00}");
+
+			//3
+			vehicles[^1].TaxCoefficient = 1.3f;
+
+			//4
+			float maxCoefficient = 0;
+			for (int i = 0; i < vehicles.Length; i++)
```
Comment (aces47) [inline DevIncubator_Autopark/Program.cs:35]: You can calculate max and average coefficients in one loop.

File: DevIncubator_Autopark/VehicleType.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace DevIncubator_Autopark
+{
+	class VehicleType
```
Comment (aces47) [inline DevIncubator_Autopark/VehicleType.cs:5]: You need to write access modifiers. Always.

File: DevIncubator_Autopark/VehicleType.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+	{
+		private string _typeName;
```
Comment (aces47) [inline DevIncubator_Autopark/VehicleType.cs:7]: Better to use autoprops, I think describing props is not necessary in that task.

File: DevIncubator_Autopark/Program.cs [hunk 4]
```diff
@@ -4,9 +4,44 @@ namespace DevIncubator_Autopark
...
+			float averageCoefficient = 0;
+			for (int i = 0; i < vehicles.Length; i++)
+			{
+				vehicles[i].Display();
+
+				averageCoefficient += vehicles[i].TaxCoefficient;
+			}
+
+			averageCoefficient /= vehicles.Length;
+			Console.WriteLine($"Average tax coefficient = {averageCoefficient:0.00}");
```
Comment (PushMeNow) [inline DevIncubator_Autopark/Program.cs:28]: Create pls separate methods for calculation of avarage and max values. It's more readable. As I see these methods will accept collection of vihicles and return floating point number.

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi! You did good. But there are some moments with need to fix.
Also add /bin and /.vs folders in gitignore.

File: DevIncubator_Autopark/Program.cs (hunk 4 above)
Comment (DenisCo1100100) [reply DevIncubator_Autopark/Program.cs]: fixed

**Review by PushMeNow — APPROVED** [review]

**Pull request merged**